File: src/main/java/br/com/alfser/projeto/pagamentos/PagamentoRepository.java
package br.com.alfser.projeto.pagamentos;

import br.com.alfser.projeto.pagamentos.models.Pagamento;
import org.bson.types.ObjectId;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface PagamentoRepository extends MongoRepository<Pagamento, ObjectId> {
}

File: processa-pagamentos/src/main/kotlin/br/com/alfser/projeto/processa_pagamentos/repositories/PagamentoRepository.kt
package br.com.alfser.projeto.processa_pagamentos.repositories

import br.com.alfser.projeto.processa_pagamentos.models.Pagamento
import org.bson.types.ObjectId
import org.springframework.data.mongodb.repository.MongoRepository

interface PagamentoRepository : MongoRepository<Pagamento, ObjectId>{
    fun findAllByIsProcessadoFalse() : List<Pagamento>
}